utils/src/main/java/org/wso2/ei/b7a/jms/JmsMessageUtils.java

```diff
@@ -96,7 +96,26 @@ public static ArrayValue getJmsPropertyNames(Message message) {
         }
     }
 
-    /**
+    public static ArrayValue getBytes(Message message, String name) {
```

Comment:
This method must reside in the `JmsMapMessageUtils`.

Reply:
resolved